quinn-proto/src/connection/mod.rs

```diff
@@ -1914,6 +1915,16 @@ where
                         Ok(())
                     }
                     Header::VersionNegotiate { .. } => {
+                        if self.total_authed_packets > 1 {
```

Comment:
I think this isn't introduced until later in your commit series? 😄

Reply:
It's introduced in the immediately prior commit as far as the commit graph is concerned; github is just confusing things by sorting by date.